server/flaskr/blueprints/auth.py

```
@@ -20,24 +20,19 @@ def signup(firstName, lastName, email, password):
-        algorithm="HS256",
-    )
-
-    permissions = UserDataHandler.get_user_permissions(userId=user.id)
+    orgId = OrganizationDataHandler.check_invite(email)
+    if orgId:
+        result = UserDataHandler.add_user_to_org(user.id, orgId)
+        assert result, "Should be able to join a org when invited"
+        set_tenant(orgId)
+        permissions = UserDataHandler.get_user_permissions(userId=user.id)
+        return (
+            create_auth_response(create_token(user.id, orgId), user, True, permissions),
```

Ruff (0.15.15)

[warning] Boolean positional value in function call

(FBT003)